packages/prisma/migrations/20241212164338_create_routing_form_response/migration.sql

```diff
@@ -0,0 +1,69 @@
+CREATE OR REPLACE VIEW "RoutingFormResponse" AS
+WITH attendees_agg AS (
+  SELECT 
+    a."bookingId",
+    json_agg(
+      json_build_object(
+        'timeZone', a."timeZone",
+        'email', a.email
+      )
+    ) as attendees
+  FROM "Attendee" a
+  GROUP BY a."bookingId"
+),
+assignment_reasons_agg AS (
+  SELECT 
+    ar."bookingId",
+    json_agg(
+      json_build_object(
+        'reasonString', ar."reasonString"
+      )
+    ) as reasons
+  FROM "AssignmentReason" ar
+  GROUP BY ar."bookingId"
+)
+SELECT 
+  r.id,
+  r.response,
+  (
+    SELECT jsonb_object_agg(
+      key,
+      CASE 
+        WHEN jsonb_typeof(value->'value') = 'string' 
+        THEN jsonb_build_object(
+          'label', value->'label',
+          'value', lower((value->>'value')::text)
+        )
+        ELSE value
+      END
+    )
+    FROM jsonb_each(r.response::jsonb)
+  ) as "responseLowercase",
```

Comment:
Im not sure im a fan of having duplicated data here - especially since it seems like both are returned to the client